tests/dvbuilder/dv_builder.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package dvbuilder
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+)
+
+const (
+	dvRandomNameLength       = 12
+	PVCSizeForRegistryImport = "512Mi"
+)
+
+// DVOption is an option type for the DVBuilder
+type DVOption func(*v1beta1.DataVolume)
+
+// NewDataVolume Set up a new DataVolume with a random name, a namespace and an optional list of options
+func NewDataVolume(namespace string, options ...DVOption) *v1beta1.DataVolume {
```

Comment:
Why is namespace required? Shouldn't it be enough to specify it with the client's request?

Reply:
@xpivarc, `namespace` is a parameter in all NewBlockDataVolume* function. In this PR, I think we should focus on the builder, keeping the rest of the code as is.

Reply:
Driving the API by old design is not the best idea/excuse imho. On top of it, it is pretty trivial to adjust the code to continue working without the `namespace` parameter.

Reply:
I second the @xpivarc, it will be better to skip namespace here.